### PR TITLE
feat: port CvTooltip to Vue3

### PR DESCRIPTION
**Pull request by OlkaB**
Contributes to #1489

## What did you do?
ported CvTooltip, CvDefinitionTooltip, CvInteractiveTooltip to Vue3

## How have you tested it?
see src\components\CvTooltip\__tests__\CvTooltip.spec.js

## Were docs updated if needed?
- [x] Yes

### CODE REVIEW

File: src/components/CvTooltip/CvTooltip.stories.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    setup() {
+      return {
+        args,
+      };
+    },
+  };
+};
+
+export const TriggerSlot = TriggerSlotTemplate.bind({});
+TriggerSlot.parameters = storyParametersObject(
```
Comment (davidnixon) [inline src/components/CvTooltip/CvTooltip.stories.js:91]: This is the only one without a default tooltip text so it looks like this when I hover 
image
```diff
diff --git a/src/components/CvTooltip/CvTooltip.stories.js b/src/components/CvTooltip/CvTooltip.stories.js
index e31778ae..8d538919 100644
--- a/src/components/CvTooltip/CvTooltip.stories.js
+++ b/src/components/CvTooltip/CvTooltip.stories.js
@@ -88,6 +88,9 @@ const TriggerSlotTemplate = args => {
 };
 
 export const TriggerSlot = TriggerSlotTemplate.bind({});
+TriggerSlot.args = {
+  tip: TipText,
+};
 TriggerSlot.parameters = storyParametersObject(
   TriggerSlot.parameters,
   triggerSlotTemplate,
```

**Review by davidnixon — COMMENTED** [review]
looks great. Thanks! one small tweak please.

**Comment by OlkaB** [comment]
@davidnixon done

File: src/components/CvTooltip/consts.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export const TipAlignments = {
```
Comment (davidnixon) [inline src/components/CvTooltip/consts.js:1]: Nice. Just making a note here as it relates to #591

**Review by davidnixon — APPROVED** [review]

**Pull request merged**